lib/pages/my_reports_pages/my_reports_page.dart

```diff
@@ -789,43 +782,47 @@ class _MyReportsPageState extends State<MyReportsPage> {
         });
   }
 
-  _getData({bool letReturn = true, bool fromMap = true}) async {
+  void _getData() async {
     try {
       loadingStream.add(true);
       double? distance;
 
-      List<Report>? list = await ApiSingleton().getReportsList(
-          location!.latitude, location!.longitude,
-          radius: (distance ?? 1000 / 2).round());
+      var myData = await ApiSingleton().getReportsList();
 
-      if (list == null) {
-        list = [];
-      }
+      myData ??= [];
```

Comment:
Why this? haven't you already set myData to the output of the API report fetch?

Reply:
that's the same as 
```
if (list == null) {
        list = [];
}
```
but the dart linter showed me the following: 
- Prefer using `??=` over testing for null.dart[prefer_conditional_assignment](https://dart-lang.github.io/linter/lints/prefer_conditional_assignment.html)

Reply:
It can be null if the api response is not status code 200

Reply:
Mmm... I was thinking of returning an empty list instead of null if the status code is not 200... In the end, you're doing the same.

Or make it one liner like: `var myData = await ApiSingleton().getReportsList() ?? [];`

But it's strange having a method that can return null if that null value is of no interest and casting again to an empty list when needed to be used.

Reply:
yes, you're right. I'll refactor it